src/main/resources/embedding-providers-config.yaml

```diff
@@ -128,7 +128,47 @@ stargate:
               vector-dimension: 768
             - name: BAAI/bge-large-en-v1.5
               vector-dimension: 1024
-
+        huggingfaceDedicated:
+          # see https://huggingface.co/docs/inference-endpoints/en/supported_tasks#sentence-embeddings
+          display-name: Hugging Face (Dedicated)
+          enabled: true
+          url: https://{endpointName}.{regionName}.{cloudName}.endpoints.huggingface.cloud/embeddings
+          supported-authentications:
+            NONE:
+              enabled: false
+            HEADER:
+              enabled: true
+              tokens:
+                - accepted: x-embedding-api-key
+                  forwarded: Authorization
+            SHARED_SECRET:
+              enabled: true
+              tokens:
+                - accepted: providerKey
+                  forwarded: Authorization
+          properties:
+            max-batch-size: 32
+          models:
```

Comment:
Since we have no models, this could be used (as I suggested earlier) as indication that caller need not (... since they can not) provide a valid model id. And UI should also not ask for model given there is none to choose from.

Reply:
yes, that is the agreement with UI.
`The Portal should not ask for the model name, the API will return an empty list of options.`